### ISSUE TITLE
Feature Request: Save enumerations as strings instead of their int index value

### ISSUE THREAD

**Issue by chanon** [ncannasse/castle#41]
It would be nice if there was an option to save enumerations as strings instead of their index value. I know it is a bit less efficient, but it is more convenient in my use case.

**Comment by RDeluxe** [comment]
I agree, this can be useful ie. for Typescript enumerations (string enums).
